Operator Documentation Update for SQL Server (#29216)
DataDog/documentation: layouts/shortcodes/dbm-sqlserver-agent-setup-kubernetes.en.md

> If you have a Kubernetes cluster, use the [Datadog Cluster Agent][1] for Database Monitoring.

Reviewer: ```suggestion
If you're running a Kubernetes cluster, use the [Datadog Cluster Agent][1] to enable Database Monitoring. If cluster checks aren’t already enabled, [follow these instructions][2] to enable them before proceeding.
```

> If cluster checks are not already enabled in your Kubernetes cluster, follow the instructions to [enable cluster checks][2]:
> 

Reviewer: ```suggestion
```

### Operator

> Using the [Operator instructions in Kubernetes and Integrations][5] as a reference, follow the steps below to set up the SQL Server integration:

Reviewer: ```suggestion
Follow the steps below to set up the SQL Server integration, using the [Operator instructions in Kubernetes and Integrations][5] as a reference.
```

1. Create or update the `datadog-agent.yaml` file with the following configuration:

    ```yaml
    apiVersion: datadoghq.com/v2alpha1
    kind: DatadogAgent
    metadata:
      name: datadog
    spec:
      global:
        clusterName: <CLUSTER_NAME>
        site: <DD_SITE>
        credentials:
          apiSecret:
            secretName: datadog-agent-secret
            keyName: api-key

      features:
        clusterChecks:
          enabled: true

      override:
        nodeAgent:
          image:
            name: agent
            tag: <AGENT_VERSION>

        clusterAgent:
          extraConfd:
            configDataMap:
              sqlserver.yaml: |-

>                 cluster_check: true # Make sure to include this flag

Reviewer: ```suggestion
                cluster_check: true # Required for cluster checks
```

                init_config:
                instances:
                - host: <HOSTNAME>,<PORT>
                  username: datadog
                  password: 'ENC[datadog_user_database_password]'
                  connector: 'odbc'
                  driver: 'FreeTDS'
                  dbm: true
                  # Optional: For additional tags
                  tags:  
                    - 'service:<CUSTOM_SERVICE>'
                    - 'env:<CUSTOM_ENV>'
    ```

2. Apply the changes to the Datadog Operator using the following command:

    ```shell
    kubectl apply -f datadog-agent.yaml
    ```

### Helm

Complete the following steps to install the [Datadog Cluster Agent][1] on your Kubernetes cluster. Replace the values to match your account and environment.

1. Complete the [Datadog Agent installation instructions][3] for Helm.
2. Update your YAML configuration file (`datadog-values.yaml` in the Cluster Agent installation instructions) to include the following:
    ```yaml
    clusterAgent:
      confd:
        sqlserver.yaml: |-

>           cluster_check: true # Make sure to include this flag

Reviewer: ```suggestion
          cluster_check: true # Required for cluster checks
```
